solana/pyth2wormhole/program/src/attest.rs

```diff
@@ -140,6 +140,13 @@ impl<'b> InstructionContext<'b> for Attest<'b> {
 }
 
 pub fn attest(ctx: &ExecutionContext, accs: &mut Attest, data: AttestData) -> SoliResult<()> {
+    if !accs.config.is_active {
+        // msg instead of trace makes sure we're not silent about this in prod
+        solana_program::msg!("This attester program is disabled!");
```

Comment:
Nice. Definitely not for this PR, but doesn't it make sense to make error `trace`s msg so they appear in production as well?

Reply:
Great point. The whole idea behind `trace!` is that on-chain logging and especially string formatting is very expensive on Solana. To the point where our program exceeds the compute budget and never completes an attestation